build/xtask/src/dist.rs

```diff
@@ -2078,24 +2086,24 @@ fn binary_to_srec(
         panic!("SREC limit of 2^24 output sections exceeded");
     }
 
-    srec_out.push(srec::Record::S7(srec::Address32(entry)));
+    srec_out.push(srec::Record::S7(srec::Address32(entry as u32)));
```

Comment:
Is there anything to check that the `entry` will fit in a `u32` before this? If not, an error should be generated instead of silently ignoring that.

Reply:
Fixed in next patchset.